tests/scenarios/v2-addon-dev-test.ts

```diff
@@ -0,0 +1,209 @@
+import path from 'path';
+import { appScenarios } from './scenarios';
+import { PreparedApp, Project } from 'scenario-tester';
+import QUnit from 'qunit';
+import merge from 'lodash/merge';
+import execa from 'execa';
+
+const { module: Qmodule, test } = QUnit;
+
+appScenarios
+  .map('v2 addon can have imports of template-only components', async project => {
+    let packageJSON = {
+      keywords: ['ember-addon'],
+      exports: {
+        './*': './dist/*',
+        './addon-main.js': './config/addon-main.cjs',
+      },
+      files: ['dist', 'config/addon-main.cjs'],
+      'ember-addon': {
+        type: 'addon',
+        version: 2,
+        main: './config/addon-main.cjs',
+        'app-js': {},
+      },
+      devDependencies: {
+        '@babel/core': '7.16.12',
+        '@babel/plugin-proposal-class-properties': '7.16.7',
+        '@babel/plugin-proposal-decorators': '^7.17.2',
+        '@babel/plugin-syntax-decorators': '7.16.7',
+        '@babel/preset-env': '^7.16.11',
+        '@rollup/plugin-babel': '^5.3.0',
+        rollup: '^2.67.0',
+      },
+    };
+
+    let addon = new Project({
+      name: 'v2-addon',
+      files: {
+        'package.json': JSON.stringify(packageJSON, null, 2),
+        config: {
+          'addon-main.cjs': `
```

Comment:
The PR I merged last night added a `baseV2Addon` utility. That would let you delete code like this that is provided by the base v2 addon template.

See for example https://github.com/embroider-build/embroider/blob/2a44b737e120f4159aa4731efd9a6484cd81fdb5/tests/scenarios/fastboot-app-test.ts#L31-L64

Reply:
thanks! this really cut down the boilerplate